orca-igor/src/main/groovy/com/netflix/spinnaker/orca/igor/tasks/GetCommitsTask.groovy

```diff
@@ -150,7 +150,8 @@ class GetCommitsTask implements DiffTask {
         clusterPair?.canary?.availabilityZones?.findResult { key, value -> key == region }
       }?.canary?.amiName
     } else if (context.deploymentDetails) { // deploy asg stage
-      return context.deploymentDetails.find { it.region == region }?.amiId
+      def details = context.deploymentDetails.find { it.region == region }
+      return details?.amiId ?: details?.ami
```

Comment:
This actually highlights a bug in the amazon image finder if we're setting ami to be amiId from Rosco https://github.com/spinnaker/rosco/blob/12a41ced53f20ad8e6bfbf906b7aa0d22251ab42/rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/aws/AWSBakeHandler.groovy#L161 but ami to be the image name in https://github.com/spinnaker/orca/blob/fae90b9c751ef31f25de92b52695aaa0b1319a50/orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/aws/AmazonImageFinder.java#L129-L130 . 

Instead of amiId and ami, we should just set it to imageId, which seems to be consistent across the board